loki/transformations/inline/functions.py

```diff
@@ -235,6 +234,13 @@ def inline_statement_functions(routine):
                 spec_map[decl] = None
     routine.spec = Transformer(spec_map).visit(routine.spec)
 
+def _get_callee_result_var(callee):
```

Comment:
Even though this is not an outward facing API, could you add a short docstring please?